test/testdata/parser/kwargs_missing_comma.rb.parse-tree.exp

```diff
@@ -0,0 +1,86 @@
+Class {
```

Comment:
i find the `parse-tree` output to be really verbose. i've been using either `desugar-tree` or `parse-tree-whitequark` because it's a little easier to read

Reply:
I switched to whitequark. It does look a lot nicer!